js/apps/system/_admin/aardvark/APP/frontend/js/views/loginView.js

```diff
@@ -135,61 +119,59 @@
       event.preventDefault();
       this.clear();
 
-      var username = $('#loginUsername').val();
-      var password = $('#loginPassword').val();
+      const username = $('#loginUsername').val();
+      const password = $('#loginPassword').val();
 
       if (!username) {
         // do not send unneccessary requests if no user is given
         return;
       }
 
-      this.collection.login(username, password, this.loginCallback.bind(this, username, password));
+      this.collection.login(username, password, () => this.loginCallback(username, password));
     },
 
     loginCallback: function (username, password, error) {
-      var self = this;
-
       if (error) {
         if (self.loginCounter === 0) {
```

Comment:
```suggestion
        if (this.loginCounter === 0) {
```

Also needs to be changed now?